WLC/code_fixing/codefixer.py

```diff
@@ -1,92 +1,181 @@
 import keyword
+import logging
+from string import digits
 
 from fuzzysearch import find_near_matches
 
 KW_LIST = keyword.kwlist + ["print", "list", "dict", "set", "file", "open", "assert", "range"]
+LOGGER = logging.getLogger()
+
+SPEC_ATTR = ["doc", "name", "qualname", "module", "defaults", "code", "globals", "dict", "closure",
+             "annotations", "kwdefaults", "main"]
 
 
 class CodeFixer:
-    def __init__(self, code, indents):
+    def __init__(self, code, indents, poss_lines):
         self.code = code
         self.indents = indents
+        self.poss_lines = poss_lines
+        self.context = []
 
     def fix(self):
         fixed_lines = []
-        contextual_data = []
-        for line in self.code.splitlines():
-            new_line, contextual_data = self._fix_line(line, contextual_data)
+        lines = self.code.splitlines()
```

Comment:
Refactor the function to `self.code.split_lines()` maybe?

Reply:
That's a built-in function on strings....

Reply:
Aah, I didn't know that. Sorry!